### PR TITLE
Bump minor version in base image

### PR DESCRIPTION
**Pull request by mvpatel2000**
# What does this PR do?

Bump minor version in base image

### CODE REVIEW

File: docker/build_matrix.yaml [hunk 1]
```diff
@@ -222,20 +209,7 @@
...
-    brand=nvidiartx,driver>=510,driver<511 brand=geforce,driver>=510,driver<511 brand=geforcertx,driver>=510,driver<511
-    brand=quadro,driver>=510,driver<511 brand=quadrortx,driver>=510,driver<511 brand=titan,driver>=510,driver<511
-    brand=titanrtx,driver>=510,driver<511 brand=tesla,driver>=515,driver<516 brand=unknown,driver>=515,driver<516
-    brand=nvidia,driver>=515,driver<516 brand=nvidiartx,driver>=515,driver<516 brand=geforce,driver>=515,driver<516
-    brand=geforcertx,driver>=515,driver<516 brand=quadro,driver>=515,driver<516 brand=quadrortx,driver>=515,driver<516
-    brand=titan,driver>=515,driver<516 brand=titanrtx,driver>=515,driver<516 brand=tesla,driver>=525,driver<526
-    brand=unknown,driver>=525,driver<526 brand=nvidia,driver>=525,driver<526 brand=nvidiartx,driver>=525,driver<526
-    brand=geforce,driver>=525,driver<526 brand=geforcertx,driver>=525,driver<526 brand=quadro,driver>=525,driver<526
-    brand=quadrortx,driver>=525,driver<526 brand=titan,driver>=525,driver<526 brand=titanrtx,driver>=525,driver<526
+  NVIDIA_REQUIRE_CUDA_OVERRIDE: ''
```
Comment (dakinggg) [inline docker/build_matrix.yaml:212]: we don't need this anymore?

**Review by dakinggg — COMMENTED** [review]
Can we run tests on the images this PR produces before merging?

File: docker/build_matrix.yaml (hunk 1 above)
Comment (mvpatel2000) [reply docker/build_matrix.yaml]: Fixed

**Review by dakinggg — APPROVED** [review]

**Comment by Skylion007** [comment]
Can we bump xformers/flash attention 2 too? I need to reinstall the latest flash attention everytime now for my torch compile exps.

**Comment by mvpatel2000** [comment]
> Can we bump xformers/flash attention 2 too? I need to reinstall the latest flash attention everytime now for my torch compile exps.

Wanna open a PR with whatever version you need?

**Pull request merged**